_providers/web.de.md

```diff
@@ -47,6 +47,10 @@ server:
     username_pattern: emaillocalpart
 before_login_hint: |
   You must allow IMAP access to your account before you can login.
+after_login_hint: |
+  Note: if you have your web.de spam settings too strict, you won't receive contact requests from new people.
+  If you want to receive contact requests, you should disable the "3-Wege-Spamschutz" in the web.de settings. 
+  Read how: https://hilfe.web.de/email/spam-und-viren/spamschutz-einstellungen.html
```

Comment:
Maybe those new users won't know what contact requests are? So I'm not sure on the wording here.
Besides that the hint is good IMO.

Reply:
Might be that they don't know how exactly they work in the Delta context - but in general they probably understand that they won't see messages from new people, maybe even better than if we say "you won't see messages from new people".

Do you have a different proposal? This was the best I could think of...